scroll-animations-1/Overview.bs

```diff
@@ -530,6 +530,7 @@ spec: cssom-view-1; type: dfn;
 		dictionary ViewTimelineOptions {
 		  Element subject;
 		  ScrollAxis axis = "block";
+		  (DOMString or sequence&lt;CSSNumericValue>) inset = "auto auto";
```

Comment:
Minor nit: "auto" expands to "auto auto" per https://drafts.csswg.org/scroll-animations-1/#view-timeline-inset . Might that be a simpler default?

>  If the second value is omitted, it is set to the first.

Reply:
Sgtm